source/backend/api/Areas/DocumentGeneration/DocumentGenerationController.cs

```diff
@@ -46,7 +47,7 @@ public DocumentGenerationController(IDocumentGenerationService documentGeneratio
 
         // [HasPermission(Permissions.GenerateDocuments)]
         [Produces("application/json")]
-        [ProducesResponseType(typeof(ExternalResult<Models.Cdogs.FileTypes>), 200)]
+        [ProducesResponseType(typeof(ExternalResponse<Models.Cdogs.FileTypes>), 200)]
```

Comment:
Standardized the naming of backend classes on the API. For 
Result -> Response

Reply:
I like this, but can we try and separate automate refactoring actions like renames from potentially functional changes like moving files around and actual code changes.

What I would love to see is a PR like this broken out into:
- Automated refactor changes that don't really need a human reviewer.
- Manual changes that could involve human error, like the using changes that broke the lease functionality in the last major backend refactoring PR.

Reply:
For the most part, this made the refactor easier on the front end to understand what the model is used for. I agree with splitting it for future prs. unfortunately, I didn't think it would be necessary for this pr.